app/views/idv/fail.html.slim

```diff
@@ -5,4 +5,4 @@ h1.h3.my0 = t('idv.titles.hardfail')
 p.mb1 = t('idv.messages.hardfail')
 p.mb1 = t('idv.messages.hardfail2', hours: Figaro.env.idv_attempt_window_in_hours)
 p.mb1 = t('idv.messages.hardfail3')
-p.mb1 = t('idv.messages.hardfail4')
+p.mb1 = t('idv.messages.hardfail4', sp: @sp_name)
```

Comment:
we will not always have this variable available, so either we need to write text that works either way or have 2 versions. see https://github.com/18F/identity-idp/pull/673#discussion_r86393566